kolena/_utils/batched_load.py

```diff
@@ -79,6 +82,18 @@ def upload_data_frame_chunk(df_chunk: pd.DataFrame, load_uuid: str) -> None:
 DFType = TypeVar("DFType", bound=LoadableDataFrame)
 
 
+def upload_smart_chunk_data_frame(df: pd.DataFrame, uuid: str, rows: int = 1000) -> None:
```

Comment:
i think we should have similar naming like `upload_data_frame`, maybe `upload_data_frame_in_smart_chunks`. we might want to use `load_uuid` as args as well for consistency.

it will be great to add some comments for this func

Reply:
will do